File: Niveau 1/7 – Conditions avancées, opérateurs booléens/09 - L'espion est démasqué !.py
nbPersonne = int(input())
for i in range(nbPersonne):
    taille = int(input())
    age = int(input())
    poids = int(input())
    aCheval = int(input())
    isBruns = int(input())
    prob = 0
    if 178 <= taille <= 182:
        prob += 1
    if age >= 34:
        prob += 1
    if poids < 70:
        prob += 1
    if aCheval == 0:
        prob += 1
    if isBruns == 1:
        prob += 1
    if prob == 5:
        print("Très probable")
    elif 3 <= prob <= 4:
        print("Probable")
    elif prob == 0:
        print("Impossible")
    else:
        print("Peu probable")

File: Niveau 1/7 – Conditions avancées, opérateurs booléens/09 - L'espion est démasqué !.cpp
#include <iostream>
using namespace std;
int main()
{
  int nbPer;
  cin >> nbPer;
  for (int i = 0; i < nbPer; i++)
  {
    int taille, age, poids, cheval, bruns, nbCrit = 0;
    cin >> taille >> age >> poids >> cheval >> bruns;
    if (taille >= 178 && taille <= 182)
    {
      nbCrit++;
    }
    if (age >= 34)
    {
      nbCrit++;
    }
    if (poids < 70)
    {
      nbCrit++;
    }
    if (cheval == 0)
    {
      nbCrit++;
    }
    if (bruns == 1)
    {
      nbCrit++;
    }

    if (nbCrit == 5)
    {
      cout << "Très probable" << endl;
    }
    else if (nbCrit == 0)
    {
      cout << "Impossible" << endl;
    }
    else if (nbCrit < 3)
    {
      cout << "Peu probable" << endl;
    }
    else
    {
      cout << "Probable" << endl;
    }
  }
}